changelog.md

```diff
@@ -7,6 +7,8 @@
 
 ### Breaking changes in the standard library
 
+- `asyncdispatch.drain` now properly takes into account `selector.hasPendingOperations` and only returns once all pending async operations are guaranteed to have completed.
+- `asyncdispatch.drain` now consistently uses the passed timeout value for all iterations of the event loop, and not just the first iteration. This is more consistent with the other asyncdispatch apis, and allows `asyncdispatch.drain` to be more efficient.
```

Comment:
I don't think these can be considered a breaking change. Code shouldn't break because of these changes.

Reply:
https://xkcd.com/1172/ 🤷‍♂